rllib/utils/metrics/metrics_logger.py

```diff
@@ -736,127 +844,35 @@ def set_value(
                 clear_on_reduce=clear_on_reduce,
             )
 
-    def delete(self, *key: Tuple[str, ...], key_error: bool = True) -> None:
-        """Deletes th egiven `key` from this metrics logger's stats.
-
-        Args:
-            key: The key or key sequence (for nested location within self.stats),
-                to delete from this MetricsLogger's stats.
-            key_error: Whether to throw a KeyError if `key` cannot be found in `self`.
-
-        Raises:
-            KeyError: If `key` cannot be found in `self` AND `key_error` is True.
-        """
-        self._del_key(key, key_error)
-
-    def reduce(
-        self,
-        key: Optional[Union[str, Tuple[str, ...]]] = None,
-        *,
-        return_stats_obj: bool = True,
-    ) -> Dict:
-        """Reduces all logged values based on their settings and returns a result dict.
-
-        The returned result dict has the exact same structure as the logged keys (or
-        nested key sequences) combined. At the leafs of the returned structure are
-        either `Stats` objects (return_stats_obj=True, which is the default) or
-        primitive (non-Stats) values. In case of `return_stats_obj=True`, the returned
-        dict with Stats at the leafs can conveniently be re-used downstream for further
-        logging and reduction operations.
-
-        For example, imagine component A (e.g. an Algorithm) containing a MetricsLogger
-        and n remote components (e.g. EnvRunner  workers), each with their own
-        MetricsLogger object. Component A now calls its n remote components, each of
-        which returns an equivalent, reduced dict with `Stats` as leafs.
-        Component A can then further log these n result dicts via its own MetricsLogger:
-        `logger.merge_and_log_n_dicts([n returned result dicts from the remote
-        components])`.
+    def reset(self) -> None:
```

Comment:
Is it possible to turn over a list of keys to reset, while keeping others? e.g. resetting all performance metrics, but keeping the counters.

Reply:
Ah, yes, for that, you'd use:
```
logger.delete(*[keys_to_delete])
```